metadata/society/public_libraries.md

```diff
@@ -0,0 +1,73 @@
+# Title
+
+Utah Public Libraries
+
+## ID
+
+5999f1b5-06b0-4c34-833b-3afb2530d2fb
+
+## Brief Summary
+
+Points dataset of public and tribal libraries, library branches, and bookmobiles in Utah.
+
+## Summary
+
+Points dataset of public and tribal libraries, library branches, and bookmobiles in Utah. This dataset does not include private or university libraries.
+
+## Description
+
+### What is the dataset?
+
+This dataset includes point locations for public libraries, base locations for bookmobiles, tribal libraries, and library branches. Additional information about libraries and the services they provide can be found on the [Utah State Library Division](https://library.utah.gov/) website.
+
+### What is the purpose of the dataset?
+
+This dataset has been made available to the public for general analytic and cartographic purposes.
+
+### What does the dataset represent?
+
+Each point feature in this dataset includes the library name, type, address, city, and county. Features also include contact information for the library, such as a phone number, PO Box, or website where applicable. We also included library hours of operation where the information has been provided.
+
+### How was the dataset created?
+
+UGRC derived this dataset using tabular data from the Utah State Library Division. We maintain this dataset and update it as new information becomes available.
+
+<!--- This is my guesstimate on how this dataset was created. Do we have further information on how this dataset was created or how it is maintained? --->
```

Comment:
Yeah, IIRC we were given a table that I geocoded.

Reply:
Is that table something that we request or they provide us on some sort of schedule?

Reply:
Spoke with Zach about this, the library division sends us updated data as needed, sounded like there wasn't a specific schedule.

Reply:
:bell: this seems to have fallen into the cracks

Reply:
The answer to the question. It sounds like you got the answer offline and didn't share it. Thanks for closing the loop.